assemble/bin/accumulo-cluster

```diff
@@ -184,14 +184,14 @@ isDebug() {
 }
 
 debug() {
-  isDebug && echo "${@@P}"
+  isDebug && echo "$1"
 }
 
 debugAndRun() {
-  debug "$@"
+  # Use $* to convert all args as a single argument
+  debug "$*"
   if ! isDebug; then
-    # shellcheck disable=SC2294
-    eval "${@@P}"
+    eval "$(printf "%q " "$@")"
```

Comment:
This doesn't actually fix the problem with the quoting, because eval is the thing requiring an extra layer of quoting to work.